bin/pycbc_banksim_skymax

```diff
@@ -148,7 +145,7 @@ def get_waveform(approximant, phase_order, amplitude_order, spin_order,
             make_padded_frequency_series(hc, filter_N, delta_f=delta_f)
 
 aprs = sorted(list(set(td_approximants() + fd_approximants())))
-aprs += ['FROM_TEMPLATE_BANK']
+# aprs += ['FROM_TEMPLATE_BANK']
```

Comment:
Remove this line if we don't need it now. It was clearly some sort of hack, but I don't remember why at this point!